src-packed/validator.js

```diff
@@ -21,15 +21,16 @@ export function hasEmptyApiKey() {
 export async function getMatches(text, matches) {
     loadAppInsights();
 
+    var replacedText = checkForIgnorableBriefPhrases(text);
     var minLength = typeof config != "undefined" ? config.MIN_REVIEW_LENGTH : 15;
-    if (text.length < minLength)
+    if (replacedText.length < minLength)
```

Comment:
This adds another regex to match against each time.

It might help performance a little if we instead did something like:

```javascript
if (text.length < minLength) {
  if (ignorableBriefPhraseRegex.test(text))
    return matches; // return an empty list early
  
  // rest of the code
}
```
You'd also make `ignorableBriefPhraseRegex` a `RegExp` object in this example.

Then we'd only be running this regex on short text. What do you think?

Reply:
Thanks for the feedback! Definitely cleaner this way - updated!